src/Displayers/StaticDisplayer.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of Laravel Exceptions.
+ *
+ * (c) Graham Campbell <graham@mineuk.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace GrahamCampbell\Exceptions\Displayers;
+
+use Exception;
+use GrahamCampbell\Exceptions\Traits\InfoTrait;
+use Illuminate\Contracts\View\Factory as View;
+
+/**
+ * This is the static displayer class.
+ *
+ * @author Libern Lin <libernlin@gmail.com>
+ */
+class StaticDisplayer implements DisplayerInterface
+{
+    use InfoTrait;
+
+    /**
+     * The view.
+     *
+     * @var string
+     */
+    protected $view;
+
+    /**
+     * Create a new static displayer instance.
+     */
+    public function __construct()
+    {
+        $this->view = file_get_contents(__DIR__.'/../../views/plain.html');
```

Comment:
I think there should be a way to get the path to the published directory. As you didn't mention the view files are configureable through publishing them in the document, so I didn't do that part.